Release doc (#3957)
cloud-hypervisor/cloud-hypervisor: README.md

Direct kernel boot to userspace should work with a rootfs from most
distributions.

> Further details can be found in the [release documentation](doc/releases.md).

Reviewer: ```suggestion
Further details can be found in the [release documentation](docs/releases.md).
```

> Further status and expectation information can be infered from the [release documentation](doc/releases.md).

Reviewer 2: ```suggestion
Further details can be found in the [release documentation](doc/releases.md).
```
